session.php

```diff
@@ -9,11 +9,21 @@
 $dbnames = explode(",", $dbname);
 
 foreach ($dbnames as &$db) {
-  $conn = new mysqli($dbhost, $dbuser, $dbpass, $db);
-  $sql = "SELECT * from humans WHERE id = '".$_SESSION['id']."'";
-  $result = $conn->query($sql);
-  if ( $result->num_rows > 0 ) {
+
+   $conn = new mysqli($dbhost.":".$dbport, $dbuser, $dbpass, $db);
+   
+   // Check connection
+   if ($conn->connect_errno) {
+      echo "Failed to connect to MySQL: " . $conn->connect_error;
+      exit();
+   }
+
+   $sql = "SELECT * from humans WHERE id = '".$_SESSION['id']."'";
```

Comment:
Think this should be done with a prepared statement as injection attack possible here? Not sure if this is pervasive

Reply:
Would be pretty difficult for an attacker to mess with a session variable in order to inject, but on the principle you are right... I need to change a few SQL to use prepare statements. Will do that but this is not really part of this PR

Reply:
Depends on the source of it and how it can be manipulated. Trust no-one :-)